rs/nns/integration_tests/BUILD.bazel

```
@@ -414,6 +414,7 @@ rust_ic_test(
     env = DEV_ENV,
     proc_macro_deps = MACRO_DEPENDENCIES + MACRO_DEV_DEPENDENCIES,
     tags = [
+        "manual",
```

can you explain why this needs to change?

---

IIRC, @basvandijk removed manual when he added nns_tests_nightly. Not sure why he removed manual. I guess it prevents this from being run in nightly tests. If that's true, we should probably not re-add "manual".

I suspect Jason added this to avoid running this during development when issuing commands like

```
bazel test //rs/nns/integration_tests/...
```

Indeed, I would not want this to run in that case either.

---

@daniel-wong-dfinity-org is right, I added it when trying to run all the integration tests except this one. I didn't mean to merge this within this PR. I'm still not sure why the manual tag cannot be added, but in any case that shouldn't be mixed into this PR. Reverted.

---

Yes, it seems to me that adding manual would stop this test from being run in nightly tests. [Here](https://github.com/daniel-wong-dfinity-org/bazel-always-excludes-tests-with-manual-tag) is how I came to that conclusion.

Therefore, I think this line should be removed.

However, I have no good recommendation about how to avoid running this test via commands like

```
bazel test //foo/bar/...
```

during development. I see two (not so great) options:

1. Just let it fail. This happens quickly anyway.
2. Add `--test_tag_filters=nns_tests_nightly` to your command(s).